mobile/src/bundles/common/components/modal/modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
```

Comment:
guess we should import it from `~/bundles/common/components/components`

Reply:
When Im doing this we have like conflict, because of to Modals in the same file